Tasks/VsTest/taskinputparser.ts

```diff
@@ -97,12 +98,20 @@ function initTestConfigurations(testConfiguration: models.TestConfigurations) {
     tl._writeLine(tl.loc('testSelectorInput', testConfiguration.testSelection));
 
     testConfiguration.testDropLocation = tl.getInput('searchFolder');
+    if (!isNullOrWhitespace(testConfiguration.testDropLocation))
```

Comment:
We want don't want to change if it is null or empty. These cases are handled when they are actually used.